TrackingTools/TransientTrack/interface/TransientTrack.h

```diff
@@ -1,147 +1,161 @@
-
-    TrajectoryStateOnSurface outermostMeasurementState() const
-	{return data().outermostMeasurementState();}
-
-    TrajectoryStateOnSurface innermostMeasurementState() const
-	{return data().innermostMeasurementState();}
-
-    TrajectoryStateClosestToPoint 
-      trajectoryStateClosestToPoint( const GlobalPoint & point ) const
-	{return data().trajectoryStateClosestToPoint(point);}
-
-    TrajectoryStateOnSurface stateOnSurface(const GlobalPoint & point) const
-	{return data().stateOnSurface(point);}
-
-    TrajectoryStateClosestToPoint impactPointTSCP() const
-	{return data().impactPointTSCP();}
-
-    TrajectoryStateOnSurface impactPointState() const
-	{return data().impactPointState();}
-
-    bool impactPointStateAvailable() const
-	{return data().impactPointStateAvailable();}
-
-    TrackCharge charge() const {return data().charge();}
-
-    bool operator== (const TransientTrack & other) const
-	{return &(data()) == &(other.data());}
-    // {return (a.persistentTrackRef()==tkr_);}
-
-    bool operator< (const TransientTrack & other) const 
-	{return &(data()) < &(other.data());}
-    // {return (initialFTS.momentum().z()<a.initialFreeState().momentum().z());}
-
-    const MagneticField* field() const {return data().field();}
-
-    const BasicTransientTrack* basicTransientTrack() const {return &(data());}
-
-    const Track & track() const {return data().track();}
-
-    TrackBaseRef trackBaseRef() const {return data().trackBaseRef();}
-
-    TrajectoryStateClosestToBeamLine stateAtBeamLine() const
-	{return data().stateAtBeamLine();}
-
-// Methods forwarded to original track.
-
-    /// first iterator to RecHits
-    trackingRecHit_iterator recHitsBegin() const { return track().recHitsBegin(); }
-    /// last iterator to RecHits
-    trackingRecHit_iterator recHitsEnd() const { return track().recHitsEnd(); }
-    /// get n-th recHit
-    TrackingRecHitRef recHit( size_t i ) const { return track().recHit( i ); }
-    /// number of RecHits
-    size_t recHitsSize() const { return track().recHitsSize(); }
-    //  hit pattern
-    const HitPattern &hitPattern() const { return track().hitPattern(); }
-    /// number of hits found 
-    unsigned short numberOfValidHits() const { return track().hitPattern().numberOfValidHits(); }
-    /// number of hits lost
-    unsigned short numberOfLostHits() const { return track().hitPattern().numberOfLostHits(HitPattern::TRACK_HITS); }
-    /// chi-squared of the fit
-    double chi2() const { return track().chi2(); }
-    /// number of degrees of freedom of the fit
-    double ndof() const { return track().ndof(); }
-    /// chi-squared divided by n.d.o.f.
-    double normalizedChi2() const { return track().chi2() / track().ndof(); }
-
-    /// Make the ReferenceCountingProxy method to check validity public
-    bool isValid() const {return Base::isValid() ;}
-
-  };
-
-}
-
-#endif
+#ifndef TrackReco_TransientTrack_h
+#define TrackReco_TransientTrack_h
```

Comment:
was this file reindented in full?
github shows as if all contents have changed.
Please check, aiming to reduce edits to the minimum necessary